src/common/helpers/custom-resource-provider-helper.ts

```diff
@@ -114,7 +114,7 @@ export function buildCustomResourceProvider(props: CRProviderProps): ICRProvider
       });
 
       const customResourceFunction = new lambda.Function(this, 'CustomResourcesFunction', {
-        code: lambda.Code.fromDockerBuild(codePath),
```

Comment:
other constructs are using this helper method, and require to keep using the "fromDockerBuild" API. We can update the helper logic to support both use cases

Reply:
Updated in latest commit